google-cloud-storage/samples/acceptance/buckets_test.rb

```diff
@@ -349,6 +350,23 @@
     end
   end
 
+  describe "get bucket class and location data" do
+    bucket_name = random_bucket_name
+    location = "US"
+    storage_class = "COLDLINE"
+
+    it "get_bucket_class_and_location" do
+      storage_client.create_bucket bucket_name,
+                                   location: location,
+                                   storage_class: storage_class
+      expected_output = "Bucket #{bucket_name} default storage class is " \
```

Comment:
This is not a default class. This is the class name we're passing. Just name the wording in that case.